ReactotronConfig.js

```diff
@@ -0,0 +1,17 @@
+import Reactotron from 'reactotron-react-native';
+import { AsyncStorage } from 'react-native';
+
+Reactotron.setAsyncStorageHandler(AsyncStorage)
```

Comment:
This adds a better visibility to check SAGA/REDUX interaction without the need to rely on Redux DevTools, and the ability to "subscribe" state changes instead of use console.log here and there. Additionally, this is just used in DEV environment and not loaded when in PROD env.